packages/react-topology/src/elements/BaseNode.ts

```diff
@@ -142,6 +145,19 @@ export default class BaseNode<E extends NodeModel = NodeModel, D = any> extends
 
   setPosition(point: Point): void {
     this.position = point;
+    this.positioned = true;
+    try {
+      this.getController().fireEvent(NODE_POSITIONED_EVENT, { node: this });
+      // eslint-disable-next-line no-empty
+    } catch (e) {}
```

Comment:
A bit concerned that this can result in a lot of event notifications during the layout which may impact performance.

I wonder if we should consider a single event notification that's debounced that can announce `model changed`.  This isn't really ideal either.